build_runner/lib/src/entrypoint/test.dart

```diff
@@ -85,8 +85,11 @@ class TestCommand extends BuildRunnerCommand {
     try {
       _ensureBuildTestDependency(packageGraph);
       options = readOptions();
-      var outputLocations = options.outputLocations ?? <String, Set<String>>{};
-      outputLocations.putIfAbsent('test', () => <String>{}).add(tempPath);
+      var outputLocations = (options.outputLocations ?? <String, Set<String>>{})
+        // Build test by default.
```

Comment:
My impression was that if `''` was present in the keys that we would build everything. Having to include `test` here worries me.